test/models/subnet_test.rb

```diff
@@ -119,29 +119,6 @@ class Subnet::Test < Subnet; end
     assert_match /is used by/, subnet.errors.full_messages.join("\n")
   end
 
-  test 'smart variable matches on subnet name' do
```

Comment:
This looks like a test for the matchers, do we have something similar for smart class parameter?